qa/cephfs/begin/0-install.yaml

```diff
@@ -3,6 +3,7 @@ tasks:
       extra_packages:
         rpm:
         - python3-cephfs
+        - cephfs-shell
```

Comment:
@batrick I think this is causing failures with rhea 8.6 - /a/vshankar-2024-01-10_06:48:43-fs-wip-vshankar-testing-20240103.072409-testing-default-smithi/7511104

```
2024-01-10T07:08:44.407 INFO:teuthology.orchestra.run.smithi146.stderr:Error: Unable to find a match: cephfs-shell
```

ceph.spec.in conditional build cephfs-shell for reel version >= 9, so the pkg isn't available for 8.6.